scripts/pipeline_generator/plugin.py

```diff
@@ -54,6 +54,7 @@ class DockerPluginConfig(BaseModel):
     command: List[str] = Field(default_factory=list)
     environment: List[str] = DEFAULT_DOCKER_ENVIRONMENT_VARIBLES
     volumes: List[str] = DEFAULT_DOCKER_VOLUMES
+    shell: List[str] = ["/bin/bash", "-c"]
```

Comment:
This file contains changes from https://github.com/vllm-project/buildkite-ci/pull/49/files which would be rebased after that PR is merged to main